### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by IWorkForFood**
Add mechanism for saving logs
into the file debug.log
After stopping the containers
the file remains at the address
\\wsl$\docker-desktop\mnt\docker-desktop-disk\data\docker\volumes 
for docker-desktop on windows

### CODE REVIEW

File: bin/entrypoint.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (egorgur) [inline bin/entrypoint.sh]: Why have you changed it?

**Review by egorgur — CHANGES_REQUESTED** [review]
Why you have changed shebang line in entrypoint.sh?

File: bin/entrypoint.sh (hunk 1 above)
Comment (IWorkForFood) [reply bin/entrypoint.sh]: Excusez-moi, I changed it

**Review by egorgur — APPROVED** [review]
Great!

**Pull request merged**